app/src/organisms/ProtocolUpload/index.tsx

```diff
@@ -154,7 +158,8 @@ export function ProtocolUpload(): JSX.Element {
   const isRunInMotion =
     runStatus === RUN_STATUS_RUNNING ||
     runStatus === RUN_STATUS_PAUSED ||
-    runStatus === RUN_STATUS_PAUSE_REQUESTED
+    runStatus === RUN_STATUS_PAUSE_REQUESTED ||
+    runStatus === RUN_STATUS_FINISHING
 
   let titleBarProps
   if (isProtocolRunLoaded && !isRunInMotion) {
```

Comment:
Looks like when the run status is `Stop Requested` the protocol page is still rendering a Close button and never goes to the else if statement. You can add in a check for the run status in the first if statement to get around this.

```suggestion
  if (isProtocolRunLoaded && !isRunInMotion && runStatus !== RUN_STATUS_STOP_REQUESTED) {
```